hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/RDBBatchOperation.java

```diff
@@ -166,21 +167,30 @@ public void close() {
    * Delete operation to be applied to a {@link ColumnFamily} batch.
    */
   private static final class DeleteOp extends Op {
-    private final byte[] key;
+    private final CodecBuffer key;
```

Comment:
DeleteOp.key is the same as Op.keyBytes.  We should move DeleteOp.key to Op, instead of adding it for all the subclasses.  (Since there were byte[] and CodecBuffer, I did not mention this previously.

Reply:
This won't hold true for delete range. Key would be null in that case

Reply:
For for delete range, just use key as startKey.

Reply:
ok